src/lib/react/VideoView.tsx

```diff
@@ -1,16 +1,47 @@
 import React, { useEffect, useRef } from "react";
+import debounce from "../utils/debounce";
+import RoomConnection from "../RoomConnection";
 
 interface VideoViewSelfProps {
     stream: MediaStream;
+    roomConnection?: RoomConnection;
+    muted?: boolean;
     style?: React.CSSProperties;
 }
 
 type VideoViewProps = VideoViewSelfProps &
     React.DetailedHTMLProps<React.VideoHTMLAttributes<HTMLVideoElement>, HTMLVideoElement>;
 
-export default ({ muted, stream, ...rest }: VideoViewProps) => {
+export default ({ muted, stream, roomConnection, ...rest }: VideoViewProps) => {
```

Comment:
One alternative to passing `roomConnection` here would be to support a `onResize` callback (still debounced). Then the VideoView is only responsible for calling back up whenever it size changes.